mage_ai/ai/ai_client.py

```diff
@@ -24,3 +24,22 @@ async def inference_with_prompt(
                 }
         """
         raise Exception('Subclasses must override this method.')
+
+    async def find_block_params(
+            self,
+            block_description: str
+    ):
+        """
+        Based on the block descriptionn, find the block type,
+        block language, pipeline type, and config for this block.
+
+        Args:
+          block_description: description of the block functions.
+
+        Returns:
```

Comment:
I would recommend returning a dictionary with these key values, which will make it easy to extend.

Reply:
Make total sense. Updated it to return dictionary instead.